### PR TITLE
Fix swagger-gen validation

### PR DESCRIPTION
**Pull request by dnephin**
Fixes the missing validation that caused #29045

cc @vieux , @bfirsh

### CODE REVIEW

**Review by LK4D4 — APPROVED** [review]
LGTM

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: hack/validate/swagger-gen [hunk 1]
```diff
@@ -4,7 +4,7 @@ export SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPTDIR}/.validate"
 
 IFS=$'\n'
-files=( $(validate_diff --diff-filter=ACMR --name-only -- 'api/types/' || true) )
+files=( $(validate_diff --diff-filter=ACMR --name-only -- 'api/types/' 'api/swagger.yaml' || true) )
```
Comment (thaJeztah) [inline hack/validate/swagger-gen]: do we need the api/swagger-gen.yml as well here?

Comment (dnephin) [inline hack/validate/swagger-gen]: I guess that would be a good idea.

**Pull request merged**